openlane/steps/openroad.py

```diff
@@ -127,6 +127,22 @@ def pdn_macro_migrator(x):
         return [x.strip()]
 
 
+DesignFormat(
+    "odb",
+    "odb",
+    "OpenDB Database",
+    "ODB",
+).register()
+
+DesignFormat(
+    "openroad_lef",
+    "ord.lef",
```

Comment:
I think similar to `npc`, `ord` is a bit ambiguous. In general, it's best if we avoid unknown acronyms

Reply:
Not aligned, it'd make the filename too damn long and OpenROAD already uses ord: https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/include/ord

Reply:
disagree but not a deal breaker.